worklist/views.py

```diff
@@ -93,11 +141,16 @@ def search_worklist(request):
 
         worklists.append(worklist)
 
-    return render(request, 'show-worklist.html', {'results': worklists})
+    return render(request, 'show-worklist.html', {'results': worklists, 'logged_in_user': logged_in_user})
 
 
 # View to search a task by it's name
 def search_task(request):
+    if request.user.is_authenticated:
+        logged_in_user = request.user.username
+    else:
+        logged_in_user = None
```

Comment:
Not a big deal, but might be a good idea to do this from a decorator so we don't repeat this logic across many views:

```python
def with_logged_in_user(view):
  @functools.wraps
  def inner(request):
    user_or_none = request.user.username if request.user.is_authenticated else None
    return view(request, user_or_none)

@with_logged_in_user
def search_task(request, user):
  ...
```

Reply:
Nice! I've made the changes.